arches_search/utils/advanced_search/search_model_registry.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, Set, Optional
+from django.db.models import UUIDField, QuerySet
```

Comment:
remove unused imports

```suggestion
from typing import Any, Dict, Set
from django.db.models import UUIDField
```